Source/WebCore/platform/CommonAtomStrings.h

```diff
@@ -42,6 +42,7 @@ namespace WebCore {
     macro(false, "false") \
     macro(lazy, "lazy") \
     macro(main, "main") \
+    macro(manual, "manual") \
```

Comment:
Why is this change needed?

Reply:
The popover getter uses it, I expect popover="manual" to be frequently used (once popover is used more widely), so I would prefer putting it in common atom strings rather than initializing a custom atom string.